packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/PropertyProvider.cs

```diff
@@ -8,10 +8,10 @@
 using Microsoft.Generator.CSharp.Input;
 using Microsoft.Generator.CSharp.Snippets;
 
-namespace Microsoft.Generator.CSharp
+namespace Microsoft.Generator.CSharp.Providers
 {
     [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
-    public class PropertyDeclaration
+    public class PropertyProvider
```

Comment:
This should be sealed

Reply:
marking it sealed for now because we can always undo this, but I think we will want to let customers override the behavior of the providers.  Similar to TypeProvider